xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1067,7 +1071,12 @@ bool CGUIWindowVideoBase::OnContextButton(int itemNumber, CONTEXT_BUTTON button)
         // any other select actions but play or resume, resume, play or playpart
         // don't make any sense here since the user already decided that he'd
         // like to play the item (just with a specific player)
-        VideoSelectAction selectAction = (VideoSelectAction)CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(CSettings::SETTING_MYVIDEOS_SELECTACTION);
+        VideoSelectAction selectAction;
+        if (item->GetVideoInfoTag()->m_type == MediaTypeEpisode)
+            selectAction = (VideoSelectAction)CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(CSettings::SETTING_MYVIDEOS_SELECTEPISODEACTION);
```

Comment:
Coding guidelines! This line is way to long. Use clang-format to get the formatting right.
And please replace the c-style cast with a c++ static cast.